### PR TITLE
feat(schema.prisma): add ServiceUser table to DB

### PR DESCRIPTION
**Pull request by Philip-Philippou1**
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/applications-service-api/prisma/schema.prisma [hunk 1]
```diff
@@ -74,6 +74,24 @@ model Representation {
...
 
+model ServiceUser {
+  id               Int      @unique @default(autoincrement())
+  serviceUserId    Int      @id
+  caseReference    String?
+  firstName        String?
+  middleName       String?
+  lastName         String?
+  email            String?
+  webAddress       String?
```
Comment (JamesP-KC) [inline packages/applications-service-api/prisma/schema.prisma:85]: this is named `website` in the Back Office db. Should we keep it consistent across BO db -> message -> FO db?

File: packages/applications-service-api/prisma/schema.prisma [hunk 2]
```diff
@@ -74,6 +74,24 @@ model Representation {
...
+  phoneNumber      String?
+  organisationName String?
+  organisationType String?
+  role             String?
```
Comment (JamesP-KC) [inline packages/applications-service-api/prisma/schema.prisma:89]: This is `jobTitle` in BO db. Same question as above regarding consistency across services

File: packages/applications-service-api/prisma/schema.prisma [hunk 3]
```diff
@@ -74,6 +74,24 @@ model Representation {
...
+  serviceUserId    Int      @id
+  caseReference    String?
+  firstName        String?
+  middleName       String?
+  lastName         String?
+  email            String?
+  webAddress       String?
+  phoneNumber      String?
+  organisationName String?
+  organisationType String?
```
Comment (JamesP-KC) [inline packages/applications-service-api/prisma/schema.prisma:88]: I don't think `organisationType` is captured anywhere in BO. We might not need it here

File: packages/applications-service-api/prisma/schema.prisma (hunk 1 above)
Comment (Philip-Philippou1) [reply packages/applications-service-api/prisma/schema.prisma:85]: For all of these I've based it off the message which isn't consistent with BO DB.

Perhaps I'll change to be consistent with BO DB.

File: packages/applications-service-api/prisma/schema.prisma (hunk 3 above)
Comment (Philip-Philippou1) [reply packages/applications-service-api/prisma/schema.prisma:88]: Good point, we don't use this, will remove.

**Review by JamesP-KC — APPROVED** [review]

**Pull request merged**